src/js/dom7-utils.js

```diff
@@ -1,6 +1,6 @@
-// DOM Library Utilites
+// DOM Library Utilities
 $.parseUrlQuery= function (url) {
-    var url = url || location.href;
+    url = url || location.href;
```

Comment:
Woah now we are modifying parameters which isn't great.

How about `var toParse = url || location.href;` and continue with that?